Docs/settings.md

```diff
@@ -211,10 +212,32 @@ Login used by the basic HTTP authentication. Must be provided if `password_prote
 #### Password
 Password used by the basic HTTP authentication. Must be provided if `password_protected` is `True`
 
+#### Cors request
+If you want to allow request from external application, you'll need to enable the CORS requests settings by defining authorized origins.
+To do so, just indicated the origins that are allowed to leverage the API. The authorize values are:
+
+False to forbid CORS request.
+```
+allowed_cors_origin: False
```

Comment:
Ho my bad I just read "allow_cors_request" ... Been too quickly on this one ! :)